python_modules/dagster/dagster/_core/execution/context/asset_execution_context.py

```diff
@@ -383,6 +383,12 @@ def partition_key(self) -> str:
     def partition_keys(self) -> Sequence[str]:
         return self.op_execution_context.partition_keys
 
+    @public
+    @property
+    @_copy_docs_from_op_execution_context
+    def has_partition_key_range(self) -> bool:
```

Comment:
this method was added to op context, so we need to add it to asset context too to avoid a breaking change